soft4pes/model/machine/induction_machine.py

```diff
@@ -157,7 +115,7 @@ def get_steady_state_psir(self, psiS_mag_ref, T_ref):
         ws = -Rr * Xs * psiR_q / (D * psiR_d)
 
         # (Electrical angular) speed of the rotor
-        wr = self.w - ws
+        wr = self.par.w - ws
```

Comment:
Let's fix thr notation for the speeds:

``w_s`` synchronous (stator) electrical angular frequency
``w_r`` rotor electrical angular frequency
``w_m`` rotor mechanical angular frequency
``w_sl`` slip angular frequency

if you prefer, you can remove the hyphen, but I believe it improves readability

Reply:
Updated in the next commit. I used `ws`, `wr`, `wm` and `w_sl` to comply with the naming practices we have set